mono/profiler/proflog.c

```diff
@@ -885,7 +892,7 @@ gc_reference (MonoObject *obj, MonoClass *klass, uintptr_t size, uintptr_t num,
 	int i;
 	uintptr_t last_offset = 0;
 	//const char *name = mono_class_get_name (klass);
-	LogBuffer *logbuffer = ensure_logbuf (20 + num * 8);
+	LogBuffer *logbuffer = ensure_logbuf (1 + 10 + 10 + 10 + 10 + num * (10 + 10));
```

Comment:
How would that help? Those would be constants that are only relevant in the exact event they correspond to. All they would do is move the event size further away from the actual code that writes data to the buffer, which makes it easier again to make the mistake of not updating the event size when adding event fields.